### PR TITLE
fix: update dnt and settings

### PR DESCRIPTION
**Pull request by Skillz4Killz**

### CODE REVIEW

File: dnt.ts [hunk 1]
```diff
@@ -1,16 +1,88 @@
...
+  },
+  entryPoints: [
+    "./mod.ts",
+    {
+      name: "./rest",
+      path: "src/rest/mod.ts",
+    },
+    {
+      name: "./gateway",
+      path: "src/gateway/mod.ts",
```
Comment (itohatweb) [inline dnt.ts:34]:
```suggestion
      path: "src/ws/mod.ts",
```

File: dnt.ts [hunk 2]
```diff
@@ -1,16 +1,88 @@
...
+      "discord-api",
+      "deno",
+      "discordeno",
+    ],
+    author: "Skillz4Killz",
+    license: "Apache License 2.0",
+    bugs: {
+      url: "https://github.com/discordeno/discordeno/issues",
+    },
+    homepage: "https://github.com/discordeno/discordeno#readme",
```
Comment (itohatweb) [inline dnt.ts:85]: Maybe we should use `discordeno.mod.land` here?
```suggestion
    homepage: "https://discordeno.mod.land",
```

File: plugins/mod.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export * from "./cache/mod.ts";
```
Comment (itohatweb) [inline plugins/mod.ts:1]: Remove this file from this PR

**Review by itohatweb — CHANGES_REQUESTED** [review]

File: plugins/mod.ts (hunk 3 above)
Comment (Skillz4Killz) [reply plugins/mod.ts:1]: This is important to this pr

**Review by itohatweb — APPROVED** [review]

**Pull request merged**